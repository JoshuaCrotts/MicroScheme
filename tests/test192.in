;
; Wacky schenanigans renaming stuff
;

(define dda +)
(define tcartbus -)
(define rewop **)
(define snoc cons)
(define rac car)
(define rdc cdr)
(rdc (snoc (dda 9 8) (tcartbus 11 (rewop 2 8))))
(rac (cons (snoc 100 42) (snoc (+ 45 12) (dda 9912 4))))
