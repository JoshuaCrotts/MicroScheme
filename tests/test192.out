-245
(100 . 42)
